test/parallel/test-repl-tab-complete.js

```diff
@@ -231,6 +231,7 @@ putIn.run([
 
 testMe.complete('proxy.', common.mustCall(function(error, data) {
   assert.strictEqual(error, null);
+  assert(data instanceof Array);
```

Comment:
Any reason for not using `Array.isArray()`? I guess performance is not an issue here.